tasks/main.yml

```diff
@@ -4,7 +4,7 @@
   tags: always
 
 - name: config should not be worldwide read- or writeable | DTAG SEC Req 3.36-3
-  file: path="{{apache_server_root}}" recurse=yes mode="o-rwx" owner="{{apache_user}}" group="{{apache_group}}"
+  file: path="{{apache_server_root}}" recurse=yes mode="o-rwx" owner=root group=root
```

Comment:
Please use proper YAML-syntax. :)

```
- name: config should not be worldwide read- or writeable | DTAG SEC Req 3.36-3
    file:
      path: "{{ apache_server_root }}"     # spaces between curly braces and variable
      recurse: yes
      mode: "o-rwx"
      owner: root
      group: root
```

Reply:
I think we should keep `apache_user` and `apache_group` params and set their defaults as root. To hardcode a `root` user can be problematic if apache is to be configured in a rootless environment.